tests/monotouch-test/Foundation/UrlRequestTest.cs

```diff
@@ -38,14 +38,7 @@ public void Mutability_30744 ()
 				// that would crash on devices
 				// NSInternalInconsistencyException -[__NSCFDictionary setObject:forKey:]: mutating method sent to immutable object
 				if (Runtime.Arch == Arch.SIMULATOR) {
```

Comment:
> I've already tested both on device and simulator

with the condition enabled ?

Reply:
Hmm, good point. Seems like my device testing was totally unnecessary.

Let me remove that condition and see what happens.

Reply:
The test runs fine on device, both iOS and tvOS, with the condition removed, so I just pushed that change.